packages/lisk-p2p/src/peer.ts

```diff
@@ -214,6 +214,19 @@ export class Peer extends EventEmitter {
 		}
 	}
 
+	public dropInboundConnection(code: number = 1000, reason?: string): void {
+		if (this._inboundSocket) {
+			this._inboundSocket.destroy(code, reason);
+			this._stopListeningForRPCs(this._inboundSocket);
+		}
+	}
+
+	public dropOutboundConnection(code: number = 1000, reason?: string): void {
+		if (this._outboundSocket) {
+			this._outboundSocket.destroy(code, reason);
+		}
```

Comment:
we don't need to stop listening for RPC for outbound?

Reply:
I think we are only listening for RPC's on inbound connection [peer.ts#L142](https://github.com/LiskHQ/lisk-elements/blob/1a786675868225175bd77db05f2a91ebfb099cd3/packages/lisk-p2p/src/peer.ts#L142) and not listening for RPC on outbound.